src/server.h

```diff
@@ -470,12 +475,12 @@ class Server : public con::PeerHandler, public MapEventReceiver,
 	void handleAdminChat(const ChatEventChat *evt);
 
 	// When called, connection mutex should be locked
-	RemoteClient* getClient(u16 peer_id,ClientState state_min=CS_Active);
-	RemoteClient* getClientNoEx(u16 peer_id,ClientState state_min=CS_Active);
+	RemoteClient* getClient(SessionId peer_id,ClientState state_min=CS_Active);
+	RemoteClient* getClientNoEx(SessionId peer_id,ClientState state_min=CS_Active);
 
 	// When called, environment mutex should be locked
-	std::string getPlayerName(u16 peer_id);
-	PlayerSAO* getPlayerSAO(u16 peer_id);
+	std::string getPlayerName(SessionId peer_id);
+	PlayerSAO* getPlayerSAO(SessionId peer_id);
```

Comment:
The formatting of the `getPlayer* / getClient*` functions above needs corrections.

Reply:
my bad it's old code :p